source/fundamentals/connection/tls.txt

```diff
@@ -77,6 +77,26 @@ in the following ways:
    see the :manual:`SRV Connection Format </reference/connection-string/#srv-connection-format>`
    section in the Server manual.
 
+.. note:: Workaround for an "unsafe legacy renegotiation disabled" error
+
+   The {+driver-short+} depends on OpenSSL by default. You may encounter an "unsafe
+   legacy renegotiation disabled" error in certain environments when using OpenSSL 
+   versions 3.0 and later, due to outdated SSL proxies. If you encounter this error, 
+   you can set the ``SSL_OP_LEGACY_SERVER_CONNECT`` option to resolve, as shown in 
+   the following example: 
+
+   .. code-block:: js
+      :emphasize-lines: 6
+
+      import { MongoClient } from 'mongodb';
+      import { * as crypto } from 'crypto';
```

Comment:
```suggestion
      import crypto from 'crypto';
```

I don't believe `* as` can appear inside curly braces, and there's no need to use `* as` when there's a default export already.